### DISCUSSION TITLE
Avec l'interface pas de cote Pokerstars

### DISCUSSION THREAD

**Discussion by kreon76** [pretrehr/Sports-betting#18] [General]
Bonjour,

J'ai fait un scrap sur le football sur tous les books et bizarrement il ne retrouve pas par exemple la cote Pokerstars pour le match de Lyon : 

![image](https://user-images.githubusercontent.com/78261069/107765608-742f9580-6d32-11eb-8f87-ee9a549791a4.png)

Alors que c'est bien présent sur PKS : 
![image](https://user-images.githubusercontent.com/78261069/107765742-a640f780-6d32-11eb-841f-938def9fd66f.png)

Merci par avance
Cdt

**Comment by pretrehr** [comment]
Bonjour,
C'est un problème qui survient de manière assez aléatoire avec PokerStars lorsque l'on effectue un scrap sur plusieurs compétitions. En fait, lorsque l'on passe d'une compétition à l'autre sur PokerStars, il arrive que la nouvelle page conserve les matchs de la page précédente pendant un certain temps (qui peut aller jusqu'à plusieurs secondes), alors que la page indique pourtant que le chargement est terminé. Sur le screenshot, LaLiga est sélectionnée mais les matchs de Premier League sont encore affichés : 
![image](https://user-images.githubusercontent.com/43851883/107770049-223e3e00-6d39-11eb-8123-519698869315.png)
La solution serait donc de pouvoir reconnaître quand les matchs présents sont identiques aux matchs de la page précédente. J'y réfléchis encore.

Pour le moment, lorsque ce genre de problème arrive, je n'ai pas d'autres solutions que de vous conseiller de recommencer le scrap, cela devrait résoudre le problème.

**Reply by kreon76** [reply]
Daccord merci pour votre réponse rapide
